### ISSUE TITLE
Issues with expo init and selecting a Blank template

### ISSUE THREAD

**Issue by sammartinez** [expo/expo-cli#2964]
## Description
I am experiencing an error with the `expo init` command when selecting the `blank` template from the command interface. I have uninstalled, reinstalled, validated that I am on the latest versions of node and npm, however still experiencing this issue. 

## Expected Behavior

```
✔ What would you like to name your app? … my-app
✔ Choose a template: › blank                 a minimal app as clean as an empty canvas
...
Would continue without any errors.
```

## Observed Behavior

```
expo init
✔ What would you like to name your app? … my-app
✔ Choose a template: › blank                 a minimal app as clean as an empty canvas
arg.indexOf is not a function
TypeError: arg.indexOf is not a function
    at npa (/usr/local/lib/node_modules/expo-cli/node_modules/npm-package-arg/npa.js:30:75)
    at action (/usr/local/lib/node_modules/expo-cli/src/commands/init.ts:231:20)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
    at Command.<anonymous> (/usr/local/lib/node_modules/expo-cli/src/exp.ts:346:7)
```

## Environment

Unable to provide `expo diagnostics` as this is happening when creating a fresh application and can not get past the `blank` template step

**Comment by AdamHelal** [comment]
Hello Sam,
I had the same issue when I updated to expo v.4.

So I uninstalled and reinstalled an older version (3.28.6) using the command npm install expo-cli@3.28.6 --global.

**Comment by brentvatne** [comment]
hi there!

could you folks let me know what terminal you are using and what operating system?

**Comment by AdamHelal** [comment]
Used regular cmd on windows 10. Also tried on cmder(unix shell emulator).

**Comment by brentvatne** [comment]
ok able to repro this if i don't provide a name and it prompts for template name.

you can do `expo init my-app` and that will work, we'll fix this asap

**Comment by brentvatne** [comment]
fixed in expo-cli@4.0.2, sorry for the hassle!

**Comment by sammartinez** [comment]
No worries thank you @brentvatne !
